src/containers/Search.jsx

```diff
@@ -44,9 +44,24 @@ class Search extends Component {
       });
   };
 
+  insertionsort = (inputArr) => {
```

Comment:
create another module directory under `src` like common/ or utils/ and put this over there.
Write a jest unit test to confirm the test passes for unsorted to sorted array;